scripts/prerequisites.sh

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 
 OPSYS=$(uname)
-KUSTOMIZE_VERSION=v3.7.0
+KUSTOMIZE_VERSION=v3.8.4
```

Comment:
```suggestion
KUSTOMIZE_VERSION=v3.7.0
```

kustomize v3.8.x is not working atm.:
`The Deployment "veidemann-dns-resolver" is invalid: spec.template.spec.containers[0].ports[2].name: Duplicate value: "dns"
`